monkey/monkey_island/cc/app.py

```diff
@@ -71,9 +71,12 @@ def serve_home():
 
 def init_app_config(app, mongo_url):
     app.config['MONGO_URI'] = mongo_url
-    app.config['SECRET_KEY'] = str(uuid.getnode())
-    app.config['JWT_AUTH_URL_RULE'] = '/api/auth'
-    app.config['JWT_EXPIRATION_DELTA'] = env_singleton.env.get_auth_expiration_time()
+
+    # See https://flask-jwt-extended.readthedocs.io/en/stable/options
+    app.config['JWT_TOKEN_LOCATION'] = ['headers']
+    app.config['JWT_ACCESS_TOKEN_EXPIRES'] = env_singleton.env.get_auth_expiration_time()
+    # Invalidate the signature of JWTs between server resets.
+    app.config['JWT_SECRET_KEY'] = str(uuid.uuid4())
```

Comment:
It's stored in mongodb between resets?

Reply:
Nope - I meant Python process restarted, fixed the doc.